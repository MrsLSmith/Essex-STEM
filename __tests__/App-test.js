/* global describe, jest, expect, it, beforeEach */
// import "react-native";
// import React from "react";
// import App from "../App";
// import renderer from "react-test-renderer";
// import NavigationTestUtils from "react-navigation/NavigationTestUtils";

describe("App snapshot", () => {
    // TODO: get these tests working
    it("returns a fake test", () => expect(true).toBe(true));

    // jest.useFakeTimers();
    // beforeEach(() => {
    //     NavigationTestUtils.resetInternalState();
    // });
    //
    // it("renders the loading screen", async () => {
    //     const tree = renderer.create(<App />).toJSON();
    //     expect(tree).toMatchSnapshot();
    // });
    //
    // it("renders the root without loading screen", async () => {
    //     const tree = renderer.create(<App skipLoadingScreen={ true } />).toJSON();
    //     expect(tree).toMatchSnapshot();
    // });
});
