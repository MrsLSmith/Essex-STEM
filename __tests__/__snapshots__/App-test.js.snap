// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders the loading screen 1`] = `null`;

exports[`App snapshot renders the root without loading screen 1`] = `null`;
